src/controllers/sold.land.filter.controller.ts

```diff
@@ -16,8 +17,12 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 
     const appData: ApplicationData = getApplicationData(req.session);
 
+    let backLinkUrl: string = config.LANDING_PAGE_URL;
+    if (isActiveFeature(config.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022)) {
+      backLinkUrl = config.LANDING_PAGE_STARTING_NEW_URL;
+    }
     return res.render(config.SOLD_LAND_FILTER_PAGE, {
-      backLinkUrl: config.LANDING_PAGE_URL,
+      backLinkUrl: backLinkUrl,
```

Comment:
Same logic should be added on the navigation logic as well, something like this [here](https://github.com/companieshouse/overseas-entities-web/blob/9bab1a6c7fe9b2ca1cfec15e5d8fc76767ebc3f8/src/utils/navigation.ts#L6) but for the sold land link [here](https://github.com/companieshouse/overseas-entities-web/blob/9bab1a6c7fe9b2ca1cfec15e5d8fc76767ebc3f8/src/utils/navigation.ts#L20)